src/types.ts

```diff
@@ -104,17 +104,37 @@ export type Blockchain =
       ens?: NamingServiceSource;
       zns?: NamingServiceSource;
       cns?: NamingServiceSource;
-      web3Provider?: Web3Provider;
+      web3Provider?: Provider;
+      provider?: Provider;
     };
 
 export type API = {
   url: string;
 };
 
-export interface Web3Provider {
+export interface Provider {
```

Comment:
This is a breaking change. If you want to rename it, export an alias `Web3Provider` as well

Reply:
I added an export but never used it anywhere. I guess i can just mark it as deprecated to remove in future versions?